src/server/pfs/server/driver.go

```diff
@@ -451,136 +426,6 @@ func (d *driver) deleteRepo(txnCtx *txnenv.TransactionContext, repo *pfs.Repo, f
 	return nil
 }
 
-func (d *driver) deleteRepoSplitTransaction(ctx context.Context, repo *pfs.Repo, force bool) error {
```

Comment:
I think I added this from master recently. Did you mean to delete this?

Reply:
Yeah, this is just for V1. The solution to this problem should look different in V2 since we are moving this metadata to Postgres.